app/views/books/show.html.haml

```diff
@@ -7,10 +7,23 @@
   - @book.subjects.each do |s|
     = link_to s.name, subject_path(s.id)
   %br/
-  %strong Created Date:
-  = @book.created_at
+  %strong Publisher:
+  = link_to @book.publisher.name, action: "show", controller: 'publishers', id: @book.publisher.id
```

Comment:
Let's switch these routing paths to the conventional `publisher_path(@book.publisher)`